### ISSUE TITLE
pygobject_register_sinkfunc is deprecated

### ISSUE THREAD

**Issue by NicoHood** [whipper-team/whipper#45]
In order to prepare for never version where this gets removed, you should possibly fix it? However it still works. Using ArchLinux.

```
FreeDB identifies disc as Scooter / 20 Years Of Hardcore CD 01
** Message: pygobject_register_sinkfunc is deprecated (GstObject)
Creating output directory /home/arch/Music/Unknown/Unknown Artist/kKrMj0RknVuwLKBn1Om8hwcJbu0-
```

**Comment by RecursiveForest** [comment]
Heh. This is due to morituri having been built with some very embedded architecture dependencies, namely gobject in this case. Our eventual plan is to remove its support, but right now this harmless error message is merely warning us that an out of date module in an out of date version of python has an out of date function. :)

**Comment by ArchangeGabriel** [comment]
This should go away with gobject indeed. But definitively harmless, since Gst0.10 is not going to be updated anymore. ;)

**Comment by JoeLametta** [comment]
> Our eventual plan is to remove its support

Just to clarify: linked to #5.

**Comment by Freso** [comment]
Does this message still show since gstreamer was removed (#130)?

**Comment by JoeLametta** [comment]
> Does this message still show since gstreamer was removed (#130)?

Haven't tested but [python-gobject-2](https://packages.debian.org/en/jessie/python-gobject-2) is still present (in [whipper/extern/task/task.py](https://github.com/JoeLametta/whipper/blob/master/whipper/extern/task/task.py)) so I think the message is still showed.

**Comment by JoeLametta** [comment]
I'm not sure but this one has probably been fixed with pull request #271.
